invenio_communities/communities/records/jsonschemas/communities/communities-v1.0.0.json

```diff
@@ -60,7 +60,7 @@
         },
         "description": {
           "description": "Community description.",
-          "type": "string"
+          "type": ["string", "null"]
```

Comment:
I would not make this change because of the data in Zenodo, if possible. Otherwise, all the non required scheme should get the same change.